In [1]:
import pandas as pd   
import numpy as np  
import matplotlib.pyplot as plt
import seaborn as sns

In [2]:
gdp_df = pd.read_csv('../data/gdp_percapita.csv')
gdp_df.tail()

Country or Area  Year        Value  Value Footnotes
8214        Zimbabwe  1994  3965.730986              NaN
8215        Zimbabwe  1993  3634.750494              NaN
8216        Zimbabwe  1992  3649.891947              NaN
8217        Zimbabwe  1991  4126.405247              NaN
8218        Zimbabwe  1990  4013.299059              NaN

In [ ]:
	Country or Area	Year	Value	Value Footnotes
8214	Zimbabwe	1994	3965.730986	NaN
8215	Zimbabwe	1993	3634.750494	NaN
8216	Zimbabwe	1992	3649.891947	NaN
8217	Zimbabwe	1991	4126.405247	NaN
8218	Zimbabwe	1990	4013.299059	NaN


In [19]:
gdp_df.columns

Index(['Country or Area', 'Year', 'Value', 'Value Footnotes'], dtype='object')

Index(['Country or Area', 'Year', 'Value', 'Value Footnotes'], dtype='object')

In [ ]:
new_gdp = gdp_df.drop(columns = ['Value Footnotes'])
new_gdp.columns

Index(['Country or Area', 'Year', 'Value'], dtype='object')

In [21]:
new_gdp.columns = ['Country','Year','GDP_Per_Capita']
new_gdp.head()

Country  Year  GDP_Per_Capita
0  Afghanistan  2023     1992.424394
1  Afghanistan  2022     1981.710168
2  Afghanistan  2021     2144.166570
3  Afghanistan  2020     2769.685745
4  Afghanistan  2019     2927.245144

Country	Year	GDP_Per_Capita
0	Afghanistan	2023	1992.424394
1	Afghanistan	2022	1981.710168
2	Afghanistan	2021	2144.166570
3	Afghanistan	2020	2769.685745
4	Afghanistan	2019	2927.245144